tests/structured_conf/test_structured_config.py

```diff
@@ -493,95 +493,57 @@ def test_typed_list_value_error(self, class_type: str) -> None:
 
     def test_list_examples(self, class_type: str) -> None:
         module: Any = import_module(class_type)
-        conf = OmegaConf.structured(module.ListExamples)
+        conf0 = OmegaConf.structured(module.ListExamples)
+        conf1 = OmegaConf.structured(module.TupleExamples)
```

Comment:
try to use `@pytest.mark.parametrize()` on this function and pass in the different class name as well as the class type.